e2e/tests/loopRequest.test.ts

```diff
@@ -12,17 +12,31 @@ describe('loop request', () => {
 
   it('rmf responds to loop request', () => {
     $('[data-component=MainMenu] [data-item=Robots]').click();
-    const robotItem = $('[data-component=RobotItem]');
-    robotItem.click();
-    const position = robotItem.$('[data-role=position]').getText();
-    robotItem.$('button=Loop').click();
-    robotItem.$('input[name=numLoops]').setValue(1);
-    robotItem.$('button=Request').click();
-    robotItem.$('button=Info').click();
-    expect(robotItem.$('[data-role=position]')).not.toHaveText(position);
-  });
 
-  it('renders robot trajectory', () => {
-    expect($('[data-component=RobotTrajectory]')).toBeVisible();
+    browser.addLocatorStrategy('findAllRobots', () => {
+      return document.querySelectorAll('[data-component=RobotItem]');
+    });
+    const allRobotItems = browser.custom$$('findAllRobots', '[data-component=RobotItem]');
+
+    let getRobotLocations = () => {
```

Comment:
```suggestion
    const getRobotLocations = () => {
      const allRobotItems = browser.custom$$('findAllRobots', '[data-component=RobotItem]');
```

wdio complains of stale elements because `allRobotItems` is resolved, then switched to the commands panel, then switched back to the robots panel and used again. The test still works, I don't know what complications it brings but let's avoid that warning to be safe.